src/order/max.lean

```diff
@@ -1,31 +1,47 @@
 /-
 Copyright (c) 2014 Jeremy Avigad. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Jeremy Avigad, Yury Kudriashov
+Authors: Jeremy Avigad, Yury Kudryashov, Yaël Dillies
 -/
 import order.order_dual
 
 /-!
 # Minimal/maximal and bottom/top elements
 
-This file defines predicates for elements to be bottom/top and typeclasses saying that there are no
-minimal/maximal elements.
+This file defines predicates for elements to be minimal/maximal or bottom/top and typeclasses
+saying that there are no such elements.
 
 ## Predicates
 
 * `is_bot`: An element is *bottom* if all elements are greater than it.
 * `is_top`: An element is *top* if all elements are less than it.
+* `is_min`: An element is *minimal* if no element is strictly less than it.
+* `is_max`: An element is *maximal* if no element is strictly greater than it.
 
 ## Typeclasses
 
+* `no_bot_order`: An order without bottom elements.
+* `no_top_order`: An order without top elements.
 * `no_min_order`: An order without minimal elements.
 * `no_max_order`: An order without maximal elements.
+
+## See also
```

Comment:
I don't think "See also" is an appropriate header for this. Following the [style guidelines](https://leanprover-community.github.io/contribute/doc.html#header-comment), how about either moving this to the summary section, or with the predicates?

Reply:
Sure. I'm actually quite annoyed at the fact I have to strand those two lemmas in a totally unrelated (and less imported file) because that means people will think we don't have that `is_bot` is the same as `is_min` in an `order_bot` or in a `linear_order` and they will rewrite them before they understand that those are special cases of a codirected order.
I think I'll move `is_directed` upstream to `order.rel_classes`, where it arguably belongs, in another PR.